tests/configDictField.py

```diff
@@ -71,30 +75,31 @@ class BadDictCheck(pexConfig.Config):
 
     def testAssignment(self):
         c = Config2()
-        self.assertRaises(pexConfig.FieldValidationError, setattr, c, "d1", {3:3})
-        self.assertRaises(pexConfig.FieldValidationError, setattr, c, "d1", {"a":0})
+        self.assertRaises(pexConfig.FieldValidationError, setattr, c, "d1", {3: 3})
+        self.assertRaises(pexConfig.FieldValidationError, setattr, c, "d1", {"a": 0})
         self.assertRaises(pexConfig.FieldValidationError, setattr, c, "d1", [1.2, 3, 4])
-        c.d1 = None; c.d1 = {"a":Config1, "b":Config1()}
+        c.d1 = None
```

Comment:
This line is strange.  It's not clear to me if it's testing that you can set it to `None` or it's working around some bug or if it's just an accident.